features/ignore_original.feature

```diff
@@ -44,3 +44,19 @@ Feature: Ignore original
     And I go to "/index.html"
     When I go to "/images/fox.jpg"
     Then the status code should be "200"
+
+  Scenario: Build original when using processed and original image
+    Given a fixture app "image"
+    And our extension is enabled with:
+      """
+      config.ignore_original = true
+      """
+    And a template named "index.html.erb" with:
+      """
+      <%= image_path 'fox.jpg', optimize: true %>
+      <%= image_path 'fox.jpg', optimize: false %>
+      """
+    And the Server is running
+    And I go to "/index.html"
+    When I go to "/images/fox.jpg"
+    Then the status code should be "200"
```

Comment:
As discussed, maybe we can add the scenario when the original image tag is used before the processed image tag. So we explicitly check both versions which can have different implications.